### PR TITLE
Code refactory for Squeezenet

### LINKED ISSUE #278: [Example] Code refactor for image_classification_tflite and image_classification_onnx

### PR DESCRIPTION
**Pull request by pinzhenx**
Fix #278

### CODE REVIEW

File: examples/image_classification_onnx/camera.html [hunk 1]
```diff
@@ -97,7 +104,7 @@
       </div>
     </div>
     <canvas id="canvas" width=224 height=224></canvas>
-    <script src="../third_party/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```
Comment (Wenzhao-Xiang) [inline examples/image_classification_onnx/camera.html]: How about using the jquery file in ./third_party?

File: examples/image_classification_onnx/index.html [hunk 2]
```diff
@@ -111,7 +118,7 @@
       </div>
     </div>
     <canvas id="canvas" width=224 height=224></canvas>
-    <script src="../third_party/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```
Comment (Wenzhao-Xiang) [inline examples/image_classification_onnx/index.html]: Same as above.

File: examples/image_classification_onnx/camera.html (hunk 1 above)
Comment (pinzhenx) [reply examples/image_classification_onnx/camera.html]: OK. I will change it soon.

**Comment by pinzhenx** [comment]
@ibelem PTAL, thanks.

**Comment by ibelem** [comment]
LGTM, thanks!

**Pull request merged**